### PR TITLE
docs: VPC endpoints IPs in Mendix Cloud v4

### PR DESCRIPTION
**Pull request by Infra-Red**
Signed-off-by: Andrei Krasnitski <andrei.krasnitski@mendix.com>

### CODE REVIEW

**Review by pommi — APPROVED** [review]

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md [hunk 1]
```diff
@@ -47,6 +47,20 @@ Mendix uses the following static IP addresses. If you need further information,
...
+
+AWS API requests originate from the following IP ranges:
+
+* `10.10.161.0/24`
+* `10.10.162.0/24`
+* `10.10.163.0/24`
+
+If you apply IP restriction to your AWS IAM user or role, make sure that the IP ranges above are allowed in your policy.
+The [aws:SourceIp](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_condition-keys.html#condition-keys-sourceip) condition
+key can be used in a policy to allow principals to make requests only from within a specified IP range.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md]: We don't really have the concept of "principal" in Mendix. Can I change this to
```suggestion
key can be used in a policy to allow Mendix apps and end-users to make requests only from within a specified IP range.
```

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md [hunk 2]
```diff
@@ -47,6 +47,20 @@ Mendix uses the following static IP addresses. If you need further information,
...
+Mendix Cloud v4 uses [VPC endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html) to privately access AWS services.
+
+AWS API requests originate from the following IP ranges:
+
+* `10.10.161.0/24`
+* `10.10.162.0/24`
+* `10.10.163.0/24`
+
+If you apply IP restriction to your AWS IAM user or role, make sure that the IP ranges above are allowed in your policy.
+The [aws:SourceIp](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_condition-keys.html#condition-keys-sourceip) condition
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md]: I'm not clear on this. 

It seems to be a different use case from whitelisting the AWS IP ranges. Are we telling users that they can/should allow the Mendix Cloud V4 IP ranges, or that they should allow their end-users browser IP addresses. Or are we telling them that they could block addresses outside those ranges?

Or have I misunderstood when you say that this is about making calls _from_ an IP address, and this is still about the AWS ranges above?

**Review by MarkvanMents — COMMENTED** [review]
@Infra-Red and @pommi 
Hi Andrei and Pim,
I have a couple of questions about the intent behind this change. I read it as two different things, but they seem to be all written as a single section.
Have I misunderstood it?
Thanks.

**Comment by MarkvanMents** [comment]
Updated text following discussion with Andrei.

**Review by MarkvanMents — APPROVED** [review]
Release agreed over Slack.

**Pull request merged**